pkg/imagevector/packrd/packed-packr.go

```diff
@@ -0,0 +1,97 @@
+// +build !skippackr
+// Code generated by github.com/gobuffalo/packr/v2. DO NOT EDIT.
+
+// You can use the "packr2 clean" command to clean up this,
+// and any other packr generated files.
+package packrd
```

Comment:
package pkg/imagevector/packrd should not be needed

Reply:
Removed